stable_baselines/common/policies.py

```diff
@@ -236,41 +295,43 @@ class FeedForwardPolicy(ActorCriticPolicy):
     :param n_steps: (int) The number of steps to run for each environment
     :param n_batch: (int) The number of batch to run (n_envs * n_steps)
     :param reuse: (bool) If the policy is reusable or not
-    :param layers: ([int]) The size of the Neural network for the policy (if None, default to [64, 64])
+    :param layers: ([int]) (deprecated, use net_arch instead) The size of the Neural network for the policy
+    (if None, default to [64, 64])
+    :param net_arch: (list) Specification of the actor-critic policy network architecture (see mlp_extractor
+    documentation for details).
+    :param act_fun: the activation function to use in the neural network.
     :param cnn_extractor: (function (TensorFlow Tensor, ``**kwargs``): (TensorFlow Tensor)) the CNN feature extraction
     :param feature_extraction: (str) The feature extraction type ("cnn" or "mlp")
     :param kwargs: (dict) Extra keyword arguments for the nature CNN feature extraction
     """
 
-    def __init__(self, sess, ob_space, ac_space, n_env, n_steps, n_batch, reuse=False, layers=None,
-                 cnn_extractor=nature_cnn, feature_extraction="cnn", **kwargs):
+    def __init__(self, sess, ob_space, ac_space, n_env, n_steps, n_batch, reuse=False, layers=None, net_arch=None,
+                 act_fun=tf.tanh, cnn_extractor=nature_cnn, feature_extraction="cnn", **kwargs):
         super(FeedForwardPolicy, self).__init__(sess, ob_space, ac_space, n_env, n_steps, n_batch, n_lstm=256,
                                                 reuse=reuse, scale=(feature_extraction == "cnn"))
-        if layers is None:
-            layers = [64, 64]
+
+        if layers is not None:
+            print("Warning: usage of the `layers` parameter is deprecated! Use net_arch instead "
+                  "(it has a different semantics though).")
+            if net_arch is not None:
+                print("Warning: the new `net_arch` parameter overrides the `layers` parameter!")
```

Comment:
Use the [python warning system](https://docs.python.org/3/library/warnings.html), it will helps devs when they are trying to control their stdout and stderr:
```python
import warnings

warnings.warn("the new `net_arch` parameter overrides the `layers` parameter!", DeprecationWarning)
```